safekeeper/spec/models/MCProposerAcceptorReconfig_p2_a4_t2_l2_c3.cfg

```diff
@@ -0,0 +1,18 @@
+CONSTANTS
+NULL = NULL
+proposers = {p1, p2}
+acceptors = {a1, a2, a3, a4}
+max_term = 2
+max_entries = 2
+max_generation = 3
+SPECIFICATION Spec
+CONSTRAINT StateConstraint
+INVARIANT
+TypeOk
+ElectionSafetyFull
+LogIsMonotonic
+LogSafety
+SYMMETRY ProposerAcceptorSymmetry
```

Comment:
Should explain why log truncation can happen in this model  (`CommittedNotTruncated`) (briefly)

Reply:
I think it shouldn't, added it back. Let's see if it fires. This is a quite huge model, I wasn't able to finish checking it yet (runs got interrupted for various reasons).